### PR TITLE
Tests created for credential commands

### PR DESCRIPTION
**Pull request by openbots-su**

### CODE REVIEW

File: OpenBots.Commands/OpenBots.Commands.Credential.Tests/GetCredentialCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+        [Fact]
+        public void GetsCredential()
+        {
+            _engine = new AutomationEngineInstance(null);
+            _getCredential = new GetCredentialCommand();
+
+            string credentialName = "CommandTestCreds";
+
+            credentialName.StoreInUserVariable(_engine, "{credName}");
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Credential.Tests/GetCredentialCommandTests.cs:21]: So this test is dependent on the credential actually existing in the server. So if the db is wiped or the server url changes, it won't work anymore. I wonder if there anything we can do to mock that? @CameronHerwig any thoughts?

File: OpenBots.Commands/OpenBots.Commands.Credential/GetCredentialCommand.cs [hunk 2]
```diff
@@ -47,7 +48,11 @@ public GetCredentialCommand()
 		{
 			CommandName = "GetCredentialCommand";
 			SelectionName = "Get Credential";
-			CommandEnabled = true;          
+			CommandEnabled = true;
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Credential/GetCredentialCommand.cs:51]: I'd rather you didn't add this to the commands directly. It's already working in studio so I don't think we be reusing code like this in all the http based commands.

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.Credential/GetCredentialCommand.cs [hunk 3]
```diff
@@ -47,7 +48,11 @@ public GetCredentialCommand()
...
+			ServicePointManager.Expect100Continue = true;
+			ServicePointManager.SecurityProtocol |= (SecurityProtocolType.Ssl3 |
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Credential/GetCredentialCommand.cs:53]: I'd rather you didn't add this to the commands directly. It's already working in studio so I don't think we be reusing code like this in all the http based commands.

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.Credential/GetCredentialCommand.cs (hunk 3 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.Credential/GetCredentialCommand.cs]: Addressed in e2483ee14ec2b5b98e28cbea37e8e63e4effe7e4
Security Protocol setup code moved to a common method in OpenBots core.

File: OpenBots.Commands/OpenBots.Commands.Credential.Tests/GetCredentialCommandTests.cs (hunk 1 above)
Comment (openbots-ch) [reply OpenBots.Commands/OpenBots.Commands.Credential.Tests/GetCredentialCommandTests.cs:21]: I would say we can rely on the dependent entities existing on the Dev server. If the test fail we can do a simple sanity check on the Server. It comes with the territory for these commands.

**Review by openbots-ch — APPROVED** [review]

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ch — APPROVED** [review]

**Pull request merged**